### PR TITLE
Release MongooseIM 6.2.0

### PR DESCRIPTION
**Pull request by pawlooss1**
## Highlights

- Introduced CETS as an alternative internal database to Mnesia
- Removed Riak
- Updated many XEP implementations to the newest version
- Implemented partial support for XEP-0386: Bind 2 and XEP-0388: Extensible SASL Profile
- Support for Erlang 26
- Various improvements and fixes

### CODE REVIEW

**Comment by mongoose-im** [comment]
[elasticsearch_and_cassandra_26](https://circleci.com/gh/esl/MongooseIM/202309) / elasticsearch_and_cassandra_mnesia / c472f799fc121b28b7b8942cd4423bfd08f2a05a
Reports [root](http://esl.github.io/circleci-mim-results/s3_reports.html?prefix=PR/4188/202309/elasticsearch_and_cassandra_mnesia.26.1.2-amd64)/ [big](https://circleci-mim-results.s3.eu-central-1.amazonaws.com/PR/4188/202309/elasticsearch_and_cassandra_mnesia.26.1.2-amd64/big/ct_run.test%4077c70cd4a724.2023-12-07_15.24.34/index.html)
**OK: 368** / Failed: 0 / **User-skipped: 38** / Auto-skipped: 0

**Comment by mongoose-im** [comment]
[elasticsearch_and_cassandra_26](https://circleci.com/gh/esl/MongooseIM/202791) / elasticsearch_and_cassandra_mnesia / 2d75cc25f135d555abc68ae447e5b4578553466b
Reports [root](http://esl.github.io/circleci-mim-results/s3_reports.html?prefix=PR/4188/202791/elasticsearch_and_cassandra_mnesia.26.1.2-amd64)/ [big](https://circleci-mim-results.s3.eu-central-1.amazonaws.com/PR/4188/202791/elasticsearch_and_cassandra_mnesia.26.1.2-amd64/big/ct_run.test%40912f31304bea.2023-12-11_14.43.07/index.html)
**OK: 368** / Failed: 0 / **User-skipped: 38** / Auto-skipped: 0

**Comment by mongoose-im** [comment]
[elasticsearch_and_cassandra_26](https://circleci.com/gh/esl/MongooseIM/202835) / elasticsearch_and_cassandra_mnesia / ab906524aa86cba1be01e8e20998c9a1c1252052
Reports [root](http://esl.github.io/circleci-mim-results/s3_reports.html?prefix=PR/4188/202835/elasticsearch_and_cassandra_mnesia.26.1.2-amd64)/ [big](https://circleci-mim-results.s3.eu-central-1.amazonaws.com/PR/4188/202835/elasticsearch_and_cassandra_mnesia.26.1.2-amd64/big/ct_run.test%40255ea6423be9.2023-12-11_15.58.56/index.html)
**OK: 368** / Failed: 0 / **User-skipped: 38** / Auto-skipped: 0

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,113 @@
...
+- Put reporter init after app startup (#4085)
+- Fix slow getaddrs call in global distribution (#4086)
+- C2S fixes (#4095, #4129)
+- Fix invalid username in scram authentication (#4118)
+- Return a proper type from `mod_muc:node_cleanup_for_host_type/3` (#4122)
+- Correctly handle the case when TLS is disabled (#4150)
+- Fix error on ping timeout with stream management (#4153)
+- Update epgsql to fix an issue with Erlang/OTP 26 (#4169)
+- `mod_muc_light` config fix (#4178)
+- Cast the postgres timestamp to integer ()
```
Comment (chrzaszcz) [inline CHANGELOG.md:76]: Looks like a leftover - remove this line

**Comment by mongoose-im** [comment]
[elasticsearch_and_cassandra_26](https://circleci.com/gh/esl/MongooseIM/202942) / elasticsearch_and_cassandra_mnesia / 6c57161349a52b4a130a4bfbc86c13c9a8161453
Reports [root](http://esl.github.io/circleci-mim-results/s3_reports.html?prefix=PR/4188/202942/elasticsearch_and_cassandra_mnesia.26.1.2-amd64)/ [big](https://circleci-mim-results.s3.eu-central-1.amazonaws.com/PR/4188/202942/elasticsearch_and_cassandra_mnesia.26.1.2-amd64/big/ct_run.test%40b5b22d1c9d82.2023-12-12_09.41.07/index.html)
**OK: 368** / Failed: 0 / **User-skipped: 38** / Auto-skipped: 0

**Pull request merged**